packages/flutter/lib/src/cupertino/theme.dart

```diff
@@ -7,13 +7,24 @@ import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 
 import 'colors.dart';
+import 'icon_theme_data.dart';
 import 'text_theme.dart';
 
 export 'package:flutter/services.dart' show Brightness;
 
 // Values derived from https://developer.apple.com/design/resources/.
-const Color _kDefaultBarLightBackgroundColor = Color(0xCCF8F8F8);
-const Color _kDefaultBarDarkBackgroundColor = Color(0xB7212121);
+const _CupertinoThemeDefaults _kDefaultTheme = _CupertinoThemeDefaults(
+  null,
+  CupertinoColors.systemBlue,
+  CupertinoColors.systemBackground,
+  CupertinoDynamicColor.withBrightness(
+    color: Color(0xF0F9F9F9),
+    darkColor: Color(0xF01D1D1D),
+    // Values extracted from navigation bar. For toolbar or tabbar the dark color is 0xF0161616.
+  ),
+  CupertinoColors.systemBackground,
+  _CupertinoTextThemeDefaults(CupertinoColors.label, CupertinoColors.inactiveGray),
+);
 
 /// Applies a visual styling theme to descendant Cupertino widgets.
 ///
```

Comment:
It would be helpful if we explained here (and probably elsewhere) that the CupertinoTheme can be overriden and that the default one is provided by CupertinoApp. Along the same lines: explain that MaterialApp includes a MaterialBasedCupertinoTheme, it will be visible to Cupertino widgets created within the MaterialApp, and it can be overridden with a CupertinoTheme.

Reply:
`CupertinoApp` and `Theme` are briefly mentioned in the see also section. Expanded it a little bit:
> See also:                                                                      
>                                                                               
> * [CupertinoThemeData], specifies the theme's visual styling.                 
> * [CupertinoApp], which will automatically add a [CupertinoTheme] based on the
   value of [CupertinoApp.theme].                                              
> * [Theme], a Material theme which will automatically add a [CupertinoTheme]   
   with a [CupertinoThemeData] derived from the Material [ThemeData].